firecracker/rootfs/docker_runtime/create_rootfs_image.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+set -euo pipefail
+set -o xtrace
+
+readonly IMAGE="/tmp/rootfs.ext4"
+readonly MOUNT_POINT="/tmp/rootfs-mount"
+readonly DIST_FILE="/dist/firecracker_rootfs.ext4.tar.gz"
+
+########################################
+# Create image and mount it.
+########################################
+# make a 50mb empty file
+dd if=/dev/zero of="${IMAGE}" bs=1M count=50
+
+# format that filesystem
+/sbin/mkfs.ext4 "${IMAGE}"
+
+# make a mount-point
+mkdir "${MOUNT_POINT}"
+
+# Mount
+mount -t fuse-ext2 -o rw+ "${IMAGE}" "${MOUNT_POINT}"
+
+########################################
+# Mutate image
+########################################
+(
+    cd "${MOUNT_POINT}"
+    echo "hello :)" > hello.txt
```

Comment:
Is it possible (at this stage of all this Firecracker work) to have a test where we can spin this up under Firecracker and make assertions about the content of the filesystem?

Reply:
I think an "unsquash/mount it and run goss tests" is absolutely worth doing _but_ not as part of this PR